posts/2023-12-12-23.0.0.12.adoc

```diff
@@ -0,0 +1,533 @@
+
+* link:https://github.com/eclipse/microprofile-metrics[MicroProfile Metrics repo]
+* link:{url-prefix}/docs/latest/introduction-monitoring-metrics.html[Monitoring with metrics documentation]
+
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC>
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // //
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/26945
+// Contact/Reviewer: yasmin-aumeeruddy
+// // // // // // // //
+[#mpt11]
+=== Expanded Java EE and Jakarta EE support in MicroProfile Telemetry 1.1
+
+MicroProfile Telemetry 1.1 provides you with the latest Open Telemetry technology as the feature now consumes OpenTelemetry-1.29.0, which is updated from 1.19.0.
+
+The feature is compatible with the following programming model combinations:
+
+* Java EE 7 paired with MicroProfile 1.4
+* Java EE 8 paired with MicroProfile 4.1
+* Jakarta EE 9 paired with MicroProfile 5.0
+* Jakarta EE 10 paired with MicroProfile 6.1.
+
+To enable this feature, add the following feature definition to your `server.xml` file:
+
+[source,xml]
+----
+<features>
+   <feature>mpTelemetry-1.1</feature>
+</features>
+----
+
+Also, you must make third-party APIs visible for your application in the `server.xml` file:
+
+[source,xml]
+----
+<webApplication location="demo-microprofile-telemetry-inventory.war" contextRoot="/">
+    <!-- enable visibility to third party apis -->
+    <classloader apiTypeVisibility="+third-party"/>
+</webApplication>
+----
+
+[#trace]
+=== Trace incoming HTTP requests with MicroProfile Telemetry 1.0 and 1.1
+Also in Open Liberty 23.0.0.12, the MicroProfile Telemetry 1.0 and 1.1 features are enhanced to automatically trace incoming HTTP requests (static files, servlets, and JSPs).
+
+For more information about MicroProfile Telemetry, see the following links:
+
+* link:https://github.com/open-telemetry/opentelemetry-specification/blob/v1.20.0/specification/trace/api.md[Tracing API]
+* link:https://openliberty.io/docs/latest/docs/latest/microprofile-telemetry.html[Enable distributed tracing with MicroProfile Telemetry]
+
+
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC>
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // //
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/27046
+// Contact/Reviewer: abutch3r
+// // // // // // // //
+// The following excerpt for issue https://github.com/OpenLiberty/open-liberty/issues/26222 was found in 2023-09-26-23.0.0.10-beta.adoc.
+// ------ <Excerpt From Previous Post: Start> ------
+// Contact/Reviewer: Azquelt
+// // // // // // // //
+
+[#mpoa]
+== MicroProfile OpenAPI 3.1: OpenAPI doc endpoint path configuration
+
+MicroProfile OpenAPI generates and serves OpenAPI documentation for RESTful Web Services (or JAX-RS) applications that are deployed to the Liberty server. The OpenAPI documentation is served from the `/openapi` endpoint  and a user interface for browsing this documentation is served from the `/openapi/ui` endpoint.
```

Comment:
this has since been fixed in [Update 2023-12-12-23.0.0.12.adoc](https://github.com/OpenLiberty/blogs/pull/3470/commits/ae2b5f6867db1d8e5e013c3f3dabd9f06ca189f1)